### PR TITLE
Update people.json

### PR DESCRIPTION
**Pull request by pragnya-suresh**
Filled

### CODE REVIEW

**Review by skprabhanjan — CHANGES_REQUESTED** [review]
@pragnya-suresh  , Please dont add a new JSON, find your SRN and update it.
Refer #1 for details.

**Comment by pragnya-suresh** [comment]
I was unable to register before the deadline. My SRN was not listed in the
file. I am interested in joining your club. I have sent an email to Anagh
Hegde, requesting him to add my details  to your database. A friend of mine
forwarded the email describing the task, which was communicated to him from
your end. I followed the steps that were listed. I would be grateful if you
could add my details and give me an opportunity to be a part of your club.
Regret any inconvenience caused.

File: people.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
   "Name": "Christy Jacob",
   "Email": "christyjacob4@gmail.com"
  },
+  
  "01fb16ecs159": {
```
Comment (AnaghHegde) [inline people.json:35]: Remove the extra line

**Review by AnaghHegde — CHANGES_REQUESTED** [review]
Small change. Then it's good to go

**Review by AnaghHegde — APPROVED** [review]

**Pull request merged**